apps/omg/lib/omg/state/transaction.ex

```diff
@@ -48,26 +45,21 @@ defmodule OMG.State.Transaction do
 
   defmacro is_metadata(metadata) do
     quote do
-      unquote(metadata) == nil or (is_binary(unquote(metadata)) and byte_size(unquote(metadata)) == 32)
+      is_binary(unquote(metadata)) and byte_size(unquote(metadata)) == 32
```

Comment:
wouldn't just byte_size == 32 be enough?

Reply:
Ha, I always wondered and finally checked. For non-binaries it seems to throw, so I suppose it is enough

Reply:
ok I noticed there's many occurrences of this. I'd rather see this as a consistent change throughout the codebase in a separate PR.

Reply:
makes sense! #cleanup